src/utils/isDarkTheme.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * @copyright Copyright (c) 2023 Grigorii Shartsev <me@shgk.me>
+ *
+ * @author Grigorii Shartsev <me@shgk.me>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * Check if dark theme is used
+ *
+ * @return {boolean}
+ */
+function checkIfDarkTheme() {
+	// Nextcloud uses --background-invert-if-dark for dark theme filters in CSS
+	// Values:
+	// - 'invert(100%)' for dark theme
+	// - 'no' for light theme
+	return window.getComputedStyle(document.body).getPropertyValue('--background-invert-if-dark') === 'invert(100%)'
```

Comment:
@skjnldsv isnt there a better way to get the setting in the frontend?

Reply:
In theory, we can use data theme(s) attributes, but I'm not sure, if it works with custom themes from appstore

Reply:
No idea tbh :/
We should most likely provide an official way for this

Ideally it's
1. has the user picked a custom theme
2. is the system asking for dark mode

Reply:
That sounds good!